homeassistant/components/smappee/sensor.py

```diff
@@ -246,6 +250,25 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
                     )
                 )
 
+        # Add today_energy_kwh sensors for switches
+        for actuator_id, actuator in service_location.actuators.items():
+            if actuator.type == "SWITCH":
+                entities.append(
+                    SmappeeSensor(
+                        smappee_base=smappee_base,
+                        service_location=service_location,
+                        sensor="switch",
+                        attributes=[
+                            f"{actuator.name} - energy today",
+                            "mdi:power-plug",
```

Comment:
There should not be an icon defined when there is a device_class defined (set icon to `None`)

Reply:
Ok, I have removed over 10 icon usages where the device_class is defined. Can you have a look at the last commit?